streams-contrib/streams-provider-linkedin/pom.xml

```diff
@@ -138,36 +138,6 @@
             </testResource>
         </testResources>
         <plugins>
-            <plugin>
-                <groupId>org.apache.streams.plugins</groupId>
```

Comment:
What's the reason for removing this dependency?

Reply:
this plugin was added/activated in this module via copy/paste early on.  the upgraded plugin is less willing to overlook being activated when there are no schemas to generate from, so i took it out.